### PR TITLE
Home screen design refresh

### PR DESCRIPTION
**Pull request by worldeggplant**
It's a bit [WIP], but ready for test.

Left:

- [ ] Clean up
- [ ] Adding icons to Lund as well
- [ ] Add dependencies that require linking to `template/`

### CODE REVIEW

File: guide-app/src/components/screens/HomeScreen/index.js [hunk 1]
```diff
@@ -89,107 +103,100 @@ class HomeScreen extends Component<Props> {
...
-
-  renderSectionFooter = (section: {
-    title: string,
-    data: NavigationItem[],
-    category: NavigationCategory,
-    hideFooter?: boolean
-  }) => {
-    if (section.hideFooter) {
-      return null;
+    if (this.props.showLoadingSpinner) {
```
Comment (joenas) [inline guide-app/src/components/screens/HomeScreen/index.js]: Add this to deconstructuring above perhaps?

File: guide-app/src/Nav.js [hunk 2]
```diff
@@ -114,7 +120,7 @@ export default class Nav extends Component<Props> {
       <ViewContainer>
         <StatusBar
           translucent={ios}
-          barStyle="light-content"
+          barStyle="dark-content" // @TODO `barStyle` should probably be added to `Colors`
```
Comment (joenas) [inline guide-app/src/Nav.js]: On the home screen the bar is purple on Android. Tested adding a `<StatusBar barStyle="dark-content" backgroundColor={Colors.white} />`to `HomeScreen` but then map is white instead.. so we should look at that

File: guide-app/src/components/shared/ImageCard.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+const styles = StyleSheet.create({
+  container: {
+    borderRadius: 10,
+    shadowColor: "#000",
+    shadowOffset: {
+      width: 0,
+      height: 2
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
```
Comment (joenas) [inline guide-app/src/components/shared/ImageCard.js:73]: Getting these warnings: 

> View #2697 of type RCTView has a shadow set but cannot calculate shadow efficiently. Consider setting a background color to fix this, or apply the shadow to a more specific component.

Looks like it's this component but not sure what background color to apply

**Review by joenas — COMMENTED** [review]
There seems to be a bug with the bottom bar. When going to the map and then back, it doesn't appear again.

File: guide-app/src/components/screens/HomeScreen/index.js [hunk 4]
```diff
@@ -89,107 +103,100 @@ class HomeScreen extends Component<Props> {
...
+        title: cat.name,
+        data,
+        category: cat,
+        showSpinner: true,
+        hideFooter: true
+      };
+    }
+  });
+
+  const items = categories[currentHomeTab].data;
```
Comment (joenas) [inline guide-app/src/components/screens/HomeScreen/index.js]: This needs a default I believe for `currentHomeTab` when state is new?

File: guide-app/src/components/shared/ImageCard.js (hunk 3 above)
Comment (joenas) [reply guide-app/src/components/shared/ImageCard.js:73]: Added `Colors.white` for now

File: guide-app/src/Nav.js (hunk 2 above)
Comment (worldeggplant) [reply guide-app/src/Nav.js]: Sounds good, I'll look into it a bit more

File: guide-app/src/components/screens/HomeScreen/index.js (hunk 4 above)
Comment (worldeggplant) [reply guide-app/src/components/screens/HomeScreen/index.js]: currentHomeTab's default is `0` in the reducer. Do you believe there is a case when the state hasn't been populated, but the HomeScreen is rendered?

Comment (joenas) [reply guide-app/src/components/screens/HomeScreen/index.js]: It keeps happening, at least once while switching languages and sometimes at initial startup so yeah somehow.

**Comment by joenas** [comment]
> There seems to be a bug with the bottom bar. When going to the map and then back, it doesn't appear again.

Fixed that in 20ad2c286bd9ae3f2915f254397ed2a2fc98ce84

**Review by joenas — APPROVED** [review]
👍

**Pull request merged**